cmd/bclient/main.go

```diff
@@ -341,15 +370,19 @@ func doHistory(item string) {
 	switch {
 	case jsonFetchErr == bclientapi.ErrNotFound:
 		fmt.Printf("\n Item %s was not found on server %s\n", item, *server)
+		return 0
```

Comment:
This line is unneeded. but maybe it is good to explicitly say that this is not an error.